waku/v2/node/storage/migration/migrations_scripts/message/00002_addSenderTimeStamp.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE IF NOT EXISTS Message_backup (
```

Comment:
Do we expect there might be more backups in future? If so, we might want to number these like `Message_backup1` or so (user version in it?)

Reply:
This is only a temp table used to copy the content of the old Message table to the new one, this temp table gets deleted at the end of this script. Please see line 27
`DROP TABLE Message_backup;`